src/component/sidebar/index.tsx

```diff
@@ -40,6 +40,7 @@ function Sidebar(props: SidebarProps) {
   const timelineSearch: ITimelineQuery = {
     userId,
     organizationId,
+    ventureId: organizationId,
```

Comment:
My understanding is that this is our own code in the webclient only. So we could actually just get it straight right away. The only relevant part for migration is the metadata which goes over network.